apps/antalmanac/src/components/dialogs/RenameSchedule.tsx

```diff
@@ -96,7 +96,7 @@ function RenameScheduleDialog(props: ScheduleNameDialogProps) {
                 <Button onClick={handleCancel} color={isDarkMode() ? 'secondary' : 'primary'}>
                     Cancel
                 </Button>
-                <Button onClick={submitName} variant="contained" color="primary" disabled={name.trim() === ''}>
+                <Button onClick={submitName} variant="contained" color="primary" disabled={name?.trim() === ''}>
```

Comment:
I think there's two (alternative) ways of approaching this problem.

1. Use a memoized variable that safely calculates the disabled state. (It's basically the same thing, but more optimized and marginally easier to read since the operation is moved to the function body instead of the returned UI.)
```ts
const disabled = useMemo(() => {
  return name?.trim() === ''
}, [name])
```

2. Initialize the state with a default value of empty string. (Unless it's possible for the name in the schedule store to be set to null, which should honestly be prevented altogether)
```ts
const [name, setName] = useState(scheduleNames[index] ?? '')
```

The former is more robust, i.e. for dealing with edge cases more, but the latter is simpler.